### DISCUSSION TITLE
[Project Beta] Automatically Adding issues/PRs to Kanban when newly Created/Mentioned/Assigned/Review Requested

### DISCUSSION THREAD

**Discussion by akiof** [community/community#8121] [Projects and Issues]
I started to use Projects (Beta) today, mainly in order to manage my ToDos (issues/PRs) with my own due date, status, etc. in my own personal view.

Then I want some way to add issues/PRs to my Project (Beta) Kanban as soon as newly Created, Mentioned to me, Assigned to me, or Review Requested to me.
Then, I can be aware of new issues/PRs and plan the task, and set due date and other attributes to issues/PRs personally.

Can I do this kind of setting now, or function will be implemented in the near future?

**Comment by Tatskaari** [comment]
I think this is in the roadmap:
https://github.com/github/roadmap/issues/286

**Comment by akiof** [comment]
Thank you.
I see the roadmap says "new issues/PRs as they are created", but I want it to be added to project also when mentioned/assigned/review requested.
